bin/pyang

```diff
@@ -252,12 +259,22 @@ Validates the YANG module in <filename> (or stdin), and all its dependencies."""
     ctx.max_status = o.max_status
 
     # make a map of features to support, per module
+    if o.features and o.exclude_features:
+        sys.stderr.write("can't specify both --features and "
```

Comment:
I think this would be more flexible if we allow a mix of `--features` and `--exclude-features`, BUT reject if the same module is present in both.  So `--features a:foo --exclude-features b:bar` would be legal.

Reply:
That was my initial proposal :) and I'm fine with allowing them both!

But does it have to be an error if the same module is present in both? It's not ambiguous. Possible use case: someone's defined an alias with a hard-coded `--features` option, and then wants to use this alias with a variety of `--exclude-features` options. Why should it be an error to exclude one of the features that's mentioned in the alias definition?

Reply:
Sorry, then I misunderstood you.  I meant "both not present for a given module".
Anyway, if we allow both, we have to let one have higher prio.  Perhaps someone else defines an alias with hardcoded `--exclude-features`, and then wants to use this alias with a variety of `--features` options?

Reply:
True! OK, I should have realised what you meant in the first place, and I agree with "both not present for a given module" (i.e., no overlap).